### DISCUSSION TITLE
Can I use the HTML srcset attribute?

### DISCUSSION THREAD

**Discussion by PetersOtto** [datenstrom/community#571] [Ask a question] [answered]
Hi,
today I have time for a closer look to this cms. It looks like that, what I have searching for. No cookies in use, adjustable and not so huge like Wordpress. Thank you for providing yellow!

However I have a question. Is there a possibility to use "srcset" with images and as well with the slider extension?

Best regards

**Comment by markseuffert** [comment, marked as answer]
No, not out of the box. Markdown doesn't offer this feature. You can use HTML inside Markdown. You can also make your own extension and generate exactly the HTML code you want. Have a look at the following extensions:

https://github.com/annaesvensson/yellow-markdown
https://github.com/datenstrom/yellow-example-feature

Hope this helps and happy coding. 😄

**Comment by PetersOtto** [comment]
Thank you!
